R-packages/evalcast/R/baseline.R

```diff
@@ -54,7 +59,7 @@ baseline_forecaster <- function(df,
         dplyr::group_modify(~ {
             point <- .x$summed[.x$time_value == max(.x$time_value)]
             tibble::tibble(probs = covidhub_probs,
-                           quantiles = point + stats::quantile(.x$resid,
+                           quantiles = point + stats::quantile(c(.x$resid, s * .x$resid),
```

Comment:
when `s = 1` (i.e., unsymmetrized case), this changes the behavior from what it was:

```
set.seed(123)
covidhub_probs <- c(0.01, 0.025, seq(0.05, 0.95, by = 0.05), 0.975, 0.99)
a <- rnorm(23)
before <- quantile(a, probs = covidhub_probs)
after <- quantile(c(a, a), probs = covidhub_probs)
range(before - after)
```

Reply:
Oh weird.  That's surprising at first, but I guess it's due to the way quantiles get defined for quantile levels that aren't multiples of 1/n (where n is the number of data points).  Good catch, thanks!

I changed it to `s = NA` in the unsymmetrized case, and here's the code blurb to convince you it works.

```
set.seed(123)
covidhub_probs <- c(0.01, 0.025, seq(0.05, 0.95, by = 0.05), 0.975, 0.99)
a <- rnorm(23)
before <- quantile(a, probs = covidhub_probs)
after <- quantile(c(a, rep(NA, length(a))), probs = covidhub_probs, na.rm = TRUE)
range(before - after)
```